src/test/resources/fixtures/kotlin/build.gradle.kts

```diff
@@ -0,0 +1,142 @@
+import com.github.gradle.node.NodeExtension
+import com.github.gradle.node.npm.task.NpmInstallTask
+import com.github.gradle.node.npm.task.NpmTask
+import com.github.gradle.node.npm.task.NpxTask
+import com.github.gradle.node.task.NodeTask
+import com.github.gradle.node.yarn.task.YarnInstallTask
+import com.github.gradle.node.yarn.task.YarnTask
+
+plugins {
+    id("com.github.node-gradle.node")
+}
+
+configure<NodeExtension> {
+    version = "12.16.1"
+    npmVersion = ""
+    yarnVersion = ""
+    npmInstallCommand = "install"
+    distBaseUrl = "https://nodejs.org/dist"
+    download = false
+    workDir = file("${project.buildDir}/nodejs")
+    npmWorkDir = file("${project.buildDir}/npm")
+    yarnWorkDir = file("${project.buildDir}/yarn")
+    nodeModulesDir = file("${project.projectDir}")
+}
+
+val npmInstallTask = tasks.withType(NpmInstallTask::class).named("npmInstall")
```

Comment:
Given that two tasks can't have the same name, wouldn't it be simpler to just use `.named()`?

Reply:
It's just to get the right task type in the configuration lambda.

Reply:
That's right, there is only one task, but this enables me to get the right task type to avoid any wild cast in the configuration in the line below.

It may exist a cleaner way of doing that?